exec/java-exec/src/main/java/org/apache/drill/exec/rpc/AbstractDisposableUserClientConnection.java

```diff
@@ -73,6 +75,7 @@ public void sendResult(RpcOutcomeListener<Ack> listener, QueryResult result) {
 
     // Release the wait latch if the query is terminated.
     final QueryState state = result.getQueryState();
+    queryState = state.toString();
```

Comment:
Should this be labelled as `queryStateAsStr` since we already have a `state` variable in the same block's context?